contracts/DayLimit.sol

```diff
@@ -38,13 +38,13 @@ contract DayLimit is Shareable {
 
   // METHODS
 
-  // (re)sets the daily limit. needs many of the owners to confirm. doesn't alter the amount already spent today.
-  function setDailyLimit(uint _newLimit) onlymanyowners(sha3(msg.data)) external {
+  // (re)sets the daily limit. doesn't alter the amount already spent today.
+  function setDailyLimit(uint _newLimit) external {
     dailyLimit = _newLimit;
   }
 
-  // resets the amount already spent today. needs many of the owners to confirm
-  function resetSpentToday() onlymanyowners(sha3(msg.data)) external {
+  // resets the amount already spent today.
+  function resetSpentToday() external {
     spentToday = 0;
   }
```

Comment:
Ahh, really? That's inconvenient. They shouldn't be internal in MultisigWallet. Can you check if there's a way of makin them external?